service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -129,6 +130,16 @@ void listDatasets() {
     assertThat(tdrJson.get("accessLevel").asText(), is(String.valueOf(DatasetAccessLevel.OWNER)));
   }
 
+  @Test
+  void getMetadata() throws JSONException {
```

Comment:
Minor nit, I would use `throws Exception` for tests since there's (IMO) nothing to be gained by using a more specific exception type, and it requires code to be changed if the thrown exception changes.